create.html

```diff
@@ -15,18 +15,14 @@
   <body>
     <header>Create</header>
     <main>
-      <!-- Formular Ausgabe funktioniert noch nicht, Java Script in index.html fehlt -->
-      <form action="index.html" method="get">
-        <!-- Formular für Fragen -->
+      <form action="index.html">
```

Comment:
You don't need the `action` atrribute. We will show on another day how to handle form data after submit via JavaScript. :)